cloudbuild.yaml

```diff
@@ -0,0 +1,21 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 7200s
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20210722-085d930'
```

Comment:
do you need a specific version here? Would latest suffice?

Reply:
This is the latest version available (at the time of the commit). But I guess it's really not important, I can switch to `latest`.

Reply:
Updated to latest.